src/ensureChangelogHeaders.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+/**
+ * Lerna has an open issue to support header customization for changelogs.
+ * Until that is closed we can use the `version` lifecycle hook and this to
+ * accomplish the same result.
+ *
+ * https://github.com/lerna/lerna/issues/2272
+ */
+
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+
+glob.sync(path.join("packages/*/CHANGELOG.md")).forEach(file => {
+  const changelog = String(fs.readFileSync(file));
+  if (changelog.startsWith("---")) return;
+
+  const directoryPath = path.dirname(file);
+  const directoryName = path.basename(directoryPath);
+  const packageName = require(`../${directoryPath}/package.json`).name;
+
+  console.log(`Fixing header on: ${file}`);
+
+  const header = `
+---
+name: ${toTitleCase(directoryName)}
+menu: Changelog
+---
+
+# ${packageName}: Change Log
+`.trim();
+
+  fs.writeFileSync(file, changelog.replace("# Change Log", header));
+});
+
+function toTitleCase(str) {
```

Comment:
I could just use [`startCase` from lodash](https://lodash.com/docs/4.17.15#startCase) here. It's not currently a dependency of Atlantis core but, it is a dependancy of `@jobber/components`. I wasn't sure if I should bring it in for this though…

Reply:
What are your concerns with bringing in lodash into atlantis core? 

is it mostly size?

I don't think I see an issue bringing in lodash into Atlantis core.

Reply:
Yea, I did't really have a good reason. I'll bring it in. :)